main.js

```diff
@@ -0,0 +1,117 @@
+import articleService from './services/ArticleService.js';
+import productService from './services/ProductService.js';
+
+class Product {
+	name;
+	description;
+	price;
+	tags;
+	images;
+	favoriteCount;
+
+	constructor(name, description, price, tags, images, favoriteCount=0) {
+		this.name = name;
+		this.description = description;
+		this.price = price;
+		this.tags = tags;
+		this.images = images;
+		this.favoriteCount = favoriteCount;
+	}
+
+	favorite() {
```

Comment:
조금 더 자연스러운 메서드명으로 변경해주세요! 보통은 동사를 기본으로 작성한답니다.


```suggestion
	increaseFavoriteCount() {
```